uint32 toID
uint32 fromID
#bool getAll
---
#float32[] bearings
float32 bearing
